lib/renderer/api/remote.js

```diff
@@ -175,6 +175,7 @@ const proxyFunctionProperties = function (remoteMemberFunction, metaId, name) {
       return true
     },
     get: (target, property, receiver) => {
+      if (property === 'toString') return target.toString.bind(target)
```

Comment:
Would you be okay adding a second guard here:

```js
if (property === 'toString' && typeof target.toString === 'function')
```

Just to be on the safe side in case they've overridden `toString` to be a property or something else.